src/integration/integration_scheduler_scalingengine_test.go

```diff
@@ -159,3 +108,19 @@ func setPolicyDateTime(policyByte []byte) string {
 
 	return fmt.Sprintf(string(policyByte), timeZone, startTime, timeNowInTimeZone.Add(2*time.Hour).Format(dateTimeFormat))
 }
+
+func checkResponse(resp *http.Response, err error, expectedStatus int) {
+	Expect(err).NotTo(HaveOccurred())
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+	Expect(err).NotTo(HaveOccurred())
+	Expect(body).To(HaveLen(0))
```

Comment:
Will rename the method, since expecting some length when not 0 will not make sense. Can keep it same and return body but think renaming better.